meson.build

```diff
@@ -61,6 +61,9 @@ else
 endif
 conf.set('CONFIG_JSONC', json_c_dep.found(), description: 'Is json-c required?')
 
+# Check for libxnvme availability
+libxnvme_dep = dependency('xnvme', version: '>=0.2.0', required: true)
```

Comment:
Please make this dependency optional, this might pull in whole set of secondary dependencies that might not be suitable for some systems. Think of initramfs usage, different init systems (linux is not only about systemd) and constrained embedded environments.